Localization/zh_cn.json

```diff
@@ -1,1229 +1,1229 @@
+	"label.cpm.pose.vr_third_person_standing": "VR 第三人称视角（站立）",
+	"label.cpm.pose.first_person_hand": "第一人称手",
+	"label.cpm.pose.health": "生命",
+	"label.cpm.pose.hunger": "饥饿",
+	"label.cpm.pose.air": "空气",
+	"label.cpm.pose.in_menu": "与方块交互",
+	"label.cpm.pose.invisible": "隐身",
+	"label.cpm.pose.light": "光照强度",
+	"label.cpm.pose.head_rotation_yaw": "头部偏航角旋转",
+	"label.cpm.pose.head_rotation_pitch": "头部俯角仰旋转",
+	"label.cpm.pose.brush_left": "左刷",
+	"label.cpm.pose.brush_right": "右刷",
+	"label.cpm.pose.crawling": "爬行",
+	"label.cpm.dir.up": "上",
+	"label.cpm.dir.down": "下",
+	"label.cpm.dir.north": "北",
+	"label.cpm.dir.south": "南",
+	"label.cpm.dir.east": "东",
+	"label.cpm.dir.west": "西",
+	"label.cpm.rot.rot_0": "旋转：0",
+	"label.cpm.rot.rot_90": "旋转：90",
+	"label.cpm.rot.rot_180": "旋转：180",
+	"label.cpm.rot.rot_270": "旋转：270",
+	"label.cpm.skin_type.default": "默认（史蒂夫）",
+	"label.cpm.skin_type.slim": "苗条（爱丽丝）",
+	"label.cpm.skin_type.unknown": "未知",
+	"label.cpm.block_reason.block_list": "玩家被屏蔽",
+	"label.cpm.block_reason.link_owerflow": "链接文件太大",
+	"label.cpm.block_reason.config_disabled": "模型加载已禁用",
+	"label.cpm.block_reason.too_many_cubes": "块太多",
+	"label.cpm.block_reason.texture_overflow": "纹理太大",
+	"label.cpm.block_reason.uuid_lock": "被UUID锁锁定",
+	"error.cpm.crash": "CustomPlayerModels的GUI已崩溃。\\请在 GitHub 上创建Issues，并附上您的Minecraft日志。\\错误：%s",
+	"error.cpm.too_many_animations": "当前皮肤层设置的自定义动画太多。\\在 \u0027Animation\u0027 选项卡下编辑皮肤层设置。\\或者关闭某些动画的皮肤层触发设置。",
+	"error.cpm.custom_anims_not_supported": "当前的皮肤层设置不支持\\自定义姿势和手势。\\请在“动画”选项卡下的皮肤层设置中启用至少 2 个皮肤层。",
+	"error.cpm.img_load_failed": "无法加载图像文件\\%s",
+	"error.cpm.img_save_failed": "无法保存图像文件\\%s",
+	"error.cpm.unknownError": "未知错误",
+	"error.cpm.templateExportAsSkin": "模板项目不能作为皮肤导出。",
+	"error.cpm.projectNotTemplate": "此项目不是模板\\使用编辑/转换为模板",
+	"error.cpm.vanillaSkinSize": "无效的原版皮肤大小，必须为 64x64",
+	"error.cpm.paste.unknownNetworkError": "连接到粘贴站点时出现未知网络错误。\\%s",
+	"error.cpm.malformedURL": "格式错误的URL",
+	"error.cpm.unknownURL": "未知URL",
+	"error.cpm.testModel": "测试游戏内模型",
+	"error.cpm.corruptedInstall": "您的Minecraft安装已损坏\\请重新安装您的游戏\\错误：",
+	"error.paste.unknown": "未知的粘贴服务器错误",
+	"error.paste.outOfSpace": "粘贴站点空间不足。\\尝试使用“编辑/粘贴”菜单清除未使用的粘贴。",
+	"error.paste.fileNameTooLong": "文件名太长",
+	"error.paste.noSession": "没有会话",
+	"error.paste.invalidSession": "无效会话",
+	"error.paste.authFail": "无法连接到粘贴站点:\\%s",
+	"error.paste.parseError": "来自服务器的损坏响应",
+	"error.paste.fileTooBig": "文件太大",
+	"narrator.cpm.field": "文本域",
+	"key.cpm.category": "CustomPlayerModels",
+	"key.cpm.gestureMenu": "手势菜单",
+	"key.cpm.renderToggle": "切换渲染",
+	"key.cpm.qa_1": "快速访问1",
+	"key.cpm.qa_2": "快速访问2",
+	"key.cpm.qa_3": "快速访问3",
+	"key.cpm.qa_4": "快速访问4",
+	"key.cpm.qa_5": "快速访问5",
+	"key.cpm.qa_6": "快速访问6",
+	"commands.cpm.setskin.success": "设置 %s的皮肤",
+	"commands.cpm.setskin.success.force": "强制设置%s的皮肤",
+	"commands.cpm.usage": "用法：设置皮肤：cpm setskin [-f|-t] <玩家> <Base64字符串> 或重置：cpm setskin -r <玩家>",
```

Comment:
This was changed to: `Valid sub commands: setskin, safety, kick, scaling, animate, effects, detect`

Reply:
Corrected